src/databricks/labs/ucx/account.py

```diff
@@ -88,6 +88,42 @@ def sync_workspace_info(self):
                 path = f"{installation.path}/{self.SYNC_FILE_NAME}"
                 ws.workspace.upload(path, info, overwrite=True, format=ImportFormat.AUTO)
 
+    def create_account_level_groups(self):
+        """
+        Crawl all workspaces, and create account level groups if a WS local group is not present in the account
+        """
+        acc_groups = {}
+        ac_grp_ids = self._ac.groups.list(attributes="id")
+        for acc_grp_id in ac_grp_ids:
+            full_account_group = self._ac.groups.get(acc_grp_id.id)
+            acc_groups[full_account_group.display_name] = full_account_group.members
+
+        workspace_clients = self.workspace_clients()
+        all_workspace_groups = {}
+        for client in workspace_clients:
+            ws_groups_ids = client.groups.list(attributes="id")
+            for grp_id in ws_groups_ids:
+                full_workpace_group = client.groups.get(grp_id.id)
+                if full_workpace_group.display_name in all_workspace_groups:
```

Comment:
what if group A exists in workspace B and C, but has different members?

Reply:
IMO the account group should be the single source of truth. 

If acc group has 3 members, ws B group has 10 members and ws C group has 5 members, we'll log two times that the group isn't consistent at all.